assets/download/app-store-badge.svg

```diff
@@ -0,0 +1,105 @@
+<svg width="162" height="54" viewBox="0 0 162 54" version="1.1" xmlns="http://www.w3.org/2000/svg" xmlns:xlink="http://www.w3.org/1999/xlink">
```

Comment:
we could clean these up a bit with ie [svgomg](https://jakearchibald.github.io/svgomg/)

Reply:
:+1: